src/app/components/team/SmoochBot/localizables.js

```diff
@@ -119,24 +66,9 @@ We can accept most images, videos, links, text messages, and shared WhatsApp mes
 
 Contact us at *[email or other contact]* for further inquiries.`,
   },
```

Comment:
I'm under the impression that all these placeholders should go too, right?
Some of them still make reference to "Reply with 0" to go back to main menu, which seems very V1.

Also it looks like all the relevant V2 content is in `SmoochBotContentAndTranslation.js`.

The reason I believe Dani left these is because `handleChangeLanguage` in `SmoochBotComponent` seem to still use these to populate a brand new workflow/language, which also feels odd.

Anyway, this doesn't look like a showstopper, I think we can ship it and I'll add it as a item to discuss with @caiosba conceptually.

Reply:
I think the comment didn't attach correctly to the lines but I'm referring to the remaining placeholders here as a whole